### PR TITLE
chore(refactor): grab frameElement using slightly stricter logic

### PR DESCRIPTION
**Pull request by Andarist**
It's the result of the discussion here: https://github.com/dmtrKovalenko/cypress-real-events/pull/29#discussion_r588995132

### CODE REVIEW

File: src/getCypressElementCoordinates.ts [hunk 1]
```diff
@@ -80,26 +96,17 @@ function getIframesPositionShift(element: HTMLElement) {
...
-    );
+  const iframes = [];
 
+  while (currentWindow !== window.top) {
+    iframes.push(getFrameElement(currentWindow));
     currentWindow = currentWindow.parent;
   }
 
-  return iframes.reduceRight(({ frameX, frameY, frameScale }, frame) => { 
+  return iframes.reduceRight(({ frameX, frameY, frameScale }, frame) => {
```
Comment (dmtrKovalenko) [inline src/getCypressElementCoordinates.ts:106]: Why this line changed?

Comment (Andarist) [reply src/getCypressElementCoordinates.ts:106]: Oh, I haven't noticed this - it's my VScode being configured to strip trailing whitespace. I can revert this change if you want.

I can also setup Prettier in a separate PR if you like that tool.

Comment (dmtrKovalenko) [reply src/getCypressElementCoordinates.ts:106]: No worries, will do this myself :) 
I need to put more time into this project – I didn't expect a lot of external contributions here. We need better CI checks, prettier, git hooks etc ;)

**Review by dmtrKovalenko — APPROVED** [review]

**Pull request merged**